bibliography-sage.bib

```diff
@@ -1917,7 +1917,7 @@ @article{Essert2013
   note         = {http://dx.doi.org/10.2140/agt.2013.13.1531},
   number       = {3},
   pages        = {1531--1578},
-  title        = {A geometric construction of panel-regular lattices for buildings of types $Ã_2$ and $C̃_2$},
+  title        = {A geometric construction of panel-regular lattices for buildings of types $\tilde{A}_2$ and $\tilde{C}_2$},
```

Comment:
![Screenshot from 2022-02-25 17-35-26](https://user-images.githubusercontent.com/207405/155752311-7178b7e2-215a-4738-ab64-eb96f92bfbff.png)

Reply:
I changed it because the tilde was to closed to C, but my approach is worse.

Reply:
you have to wrap this in brackets, i.e. `{$\tilde{A}_2$}` and then we check.